### PR TITLE
Add simple Quickstart information for running BFG

### PR DESCRIPTION
**Pull request by berendhaan**
This adds a few one liners to the readme.md regarding setting up the project as asked in issue #191.

Could it be a possibility to use the readme.md as a datasoucre for the BFG site?
If so I could add the complete instructions to the readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
-BFG Repo-Cleaner [![Build Status](https://travis-ci.org/rtyley/bfg-repo-cleaner.svg?branch=master)](https://travis-ci.org/rtyley/bfg-repo-cleaner)
-================
+@@ -3,9 +3,19 @@ BFG Repo-Cleaner [![Build Status](https://travis-ci.org/rtyley/bfg-repo-cleaner.
```
Comment (javabrett) [inline README.md]: Looks like you've left some remnants of merge-markers here.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+
 ```
+$ java -jar bfg.jar --strip-blobs-bigger-than 100M some-big-repo.git
+```
+
 
 The BFG is a simpler, faster ([10 - 720x](https://docs.google.com/spreadsheet/ccc?key=0AsR1d5Zpes8HdER3VGU1a3dOcmVHMmtzT2dsS2xNenc) faster)
 alternative to `git-filter-branch` for cleansing bad data out of your Git repository:
-
+@@ -13,4 +23,4 @@ alternative to `git-filter-branch` for cleansing bad data out of your Git reposi
```
Comment (javabrett) [inline README.md]: ... and here.

Comment (berendhaan) [inline README.md]: Removed them in the latest version.

File: README.md (hunk 1 above)
Comment (berendhaan) [inline README.md]: Removed them in the latest version.

**Pull request closed without merging**